GOALS.md

```diff
@@ -0,0 +1,29 @@
+# Why Goals
+Stating goals (and non-goals) are intended to sharpen the join understanidng the the kube-aws product vision. Expressiong the vision by 'Goals' seem to be understandable approach.
+
+#Goals
+Kube-aws is Command Line Tool (CLI) that allow me as experienced console-user to:
+
+* to plan ant prepare my kubernetes cluster alongside need AWS resources
+* browse throgh intermediate details of the configuration (of not yet provisioned clusters)
+
+Kube-aws CLI's goal is to support different experience levels of the users in terms of 
+* kubernetes administration expertise
+* CoreOS administration expertise
+* Amazone Web Services expertise
+
+to support this goal it
+* maintanes well-documented set of default settings
+* but allows to configure every aspect of the cluster
+* enables reuse of (all) custom predefined AWS resources that are precreated in AWS
```

Comment:
Would you mind alter this a bit, like "enables reusing of the existing AWS resources whenever possible, if and only if those resources conforms to the requirements of kube-aws and/or Kubernetes" so that we can ensure that kube-aws would e.g. fail-fast when an user tried to reuse invalid/wrongly configured AWS resources which ends up with a non-functional k8s cluster?